login/login.css

```diff
@@ -0,0 +1,235 @@
+body {
+  margin: 0;
+  overflow: hidden;
+}
+
+.main {
+  min-height: 100%;
+  margin: 0 auto;
+  width: 100%;
+  display: flex;
+  align-content: center;
+  align-items: center;
+  justify-content: center;
+  
+}
+
+.bg {
+  background-image: url(images_login/bg.png);
+  -webkit-background-size: contain;
+  background-size: cover;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  width: 100%;
+  
+ 
+}
+
+.bg.no-image {
+  background-image: none;
+}
+
+.bg:after {
+  content: ' ';
+  height: 100%;
+  position: absolute;
+  width: 100%;
+  opacity: .5;
+  background: linear-gradient(to bottom right, #FF6A3A 0%, #000000 100%);
+  
+}
+
+.content {
+  margin: 0 auto;
+ 
+}
+
+.content.removed {
+  display: none;
+}
+
+.controls.hidden {
+  visibility: hidden;
+}
+
+.icon {
+  height: 200px;
+  margin: 65px auto;
+  width: 150px;
+  position: relative;
+}
+
+.icon img {
+  height: 100%;
+  width: 100%;
+}
+
+.inputs {
+ position: relative;
+
+ 
+}
+
+.login {
+ left: 13px;
+opacity: .4;
+position: absolute;
+top: 6px;
+width: 20px;
+
+}
+
+.lock {
+    left: 13px;
+	opacity: .4;
+	position: absolute;
+	top: 6px;
+	width: 20px;
+}
+
+.input {
+  border-radius: 28px;
+  border: 1px solid white;
+  box-sizing: border-box;
+  color: black;
+  display: block;
+  font-size: 16px;
+  margin-bottom: 30px;
+  outline: none !important;
+  opacity: .8;
+  padding-bottom: 15px;
+  padding-left: 90px;
+  padding-top: 10px;
+  opacity: 0.2;
+  height: 56px;
+  margin-bottom: 20px;
+  max-width: 300px;
+  outline: none !important;
+  position: relative;
+  width: 100%;
+  
+}
+
+::placeholder { 
+  color:#070707;
+  opacity: 2;
+	}
+
+.button-container {
+  left: 0;
+  margin: 0 auto;
+  position: absolute;
+  top: 0;
+  width: 100%;
+}
+
+.button {
+  background-color: #F96332;
+  border: 0;
+  border-radius: 28px;
+  display: block;
+  font-size: 16px;
+  line-height: 32px;
+  text-align:center;
+  height: 56px;
+  margin: 0 auto;
+  max-width: 260px;
+  outline: none !important;
+  position: relative;
+  top: 500px;

```

Comment:
you put your button on top of all the things in the html, to then absolute postion it at the other side of the things, why?? just rethink your html structure, with a wrong html structure you can not do magic in css